docusaurus.config.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
```

Comment:
Will the Google Analytics config arrive also in this PR? It would be great to have it from a day one of the new site.

Reply:
In the next one ;)
Fake google tag is added already here:
https://botkube-docs-c3c.pages.dev/

<img width="1456" alt="image" src="https://user-images.githubusercontent.com/14016055/187445023-c8757d93-cb8e-4dd8-9e6a-13e3624c2c42.png">